### DISCUSSION TITLE
tab markers disappear when using nested snippets

### DISCUSSION THREAD

**Discussion by kadinlz** [sleepymalc/VSCode-LaTeX-Inkscape#23] [Q&A] [answered]
Say I do a fm:
`\( $1 \) $0`
My cursor will go to 1 first then I can tab to get to 0. But if I use another snippet inside, like ";a" to "\alpha", I can no longer tab outside and have to manually arrow key out. Is there a way around this? Thanks!

**Comment by kadinlz** [comment]
I think this has to do with a conflict with vscode vim after some searching, if anyone knows how to fix please let me know :)

**Reply by sleepymalc** [reply, marked as answer]
Yes, see [here](https://github.com/draivin/hsnips/issues/78), for example.
